Formula/pdftilecut.rb

```diff
@@ -0,0 +1,23 @@
+class Pdftilecut < Formula
+  desc "Sub-divide a PDF page(s) into smaller pages so you can print them"
+  homepage "https://github.com/oxplot/pdftilecut"
+  url "git@github.com:oxplot/pdftilecut.git", using: :git, tag: "v0.4", revision: "af9ca3bff60eed47026a6e1c40b6ef85e5b36393"
+  license "BSD-3-Clause"
+
+  depends_on "autoconf" => :build
+  depends_on "autogen" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "coreutils" => :build
+  depends_on "go" => :build
+  depends_on "yasm" => :build
+
+  def install
+    system "make"
+    bin.install "bin/pdftilecut" => "pdftilecut"
+  end
+
+  test do
+    system bin/"pdftilecut", "-h"
```

Comment:
We'll need a more substantial test than this, as `--help` tests rarely ever error out when we need them to.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I understand, but I'm afraid this command does not do anything except splitting the user input pdf… Even the --version flag does not exists so I'm not sure what reliable test I can write

Reply:
I see. There should be a `pdf` file in the test fixtures (see the docs). Can you split that and test for the existence of the split pdf?

Reply:
Ok, I'll write this

Reply:
Thank you!